apps/ledger-live-desktop/src/renderer/families/bitcoin/AccountHeaderManageActions.ts

```diff
@@ -44,16 +49,20 @@ const AccountHeaderActions = ({ account, parentAccount }: Props) => {
   };
 
   return [
-    {
-      key: "Stake",
-      icon: IconCoins,
-      label: t("accounts.contextMenu.yield"),
-      event: "button_clicked2",
-      eventProperties: {
-        button: "stake",
-      },
-      onClick: stakeOnClick,
-    },
+    ...(availableOnStake
```

Comment:
it's better to move the condition on the return level

Reply:
if i move the condition to the return level and there are more actions to be passed they will be skipped based on the `stakePrograms` flag, in this way you can add another action to the array and it would be independent.

Reply:
ok, thanks for the clarification